edit_test.go

```diff
@@ -340,6 +340,21 @@ func TestEditMultipleWindows(t *testing.T) {
 		}, []string{
 			"'+  alt_example_2\nwarning: changes out of sequence\nwarning: changes out of sequence, edit result probably wrong\n",
 		}},
+
+		// u
+		// 10
+		{Range{0, 0}, "test", "u", []string{
+			contents,
+			alt_contents,
+		}, []string{}},
```

Comment:
It's preferred to write `nil` instead of `[]string{}`.

https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

Reply:
I'll fix in a subsequent CL. Thanks for noting.